### PR TITLE
model2netcdf function for LPJ-GUESS

### PR DESCRIPTION
**Pull request by istfer**
model2netcdf function for LPJ-GUESS  per #753

@araiho is up for review when she's back :)

### CODE REVIEW

File: models/lpjguess/NAMESPACE [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Generated by roxygen2 (4.0.2): do not edit by hand
 
+export(model2netcdf.LPJGUESS)
```
Comment (araiho) [inline models/lpjguess/NAMESPACE]: Why do you need export?

File: models/lpjguess/R/model2netcdf.LPJGUESS.R [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  } ### End of year loop
+  
+  
+} ### End of function
+#==================================================================================================#
+
+
+####################################################################################################
+### EOF.  End of R script file.              
+####################################################################################################
```
Comment (araiho) [inline models/lpjguess/R/model2netcdf.LPJGUESS.R]: Looks good to me. I'm surprised LPJ GUESS has fewer outputs than LINKAGES.

File: models/lpjguess/R/model2netcdf.LPJGUESS.R [hunk 3]
```diff
@@ -63,6 +63,10 @@ model2netcdf.LPJGUESS <- function(outdir, sitelat, sitelon, start_date, end_date
   ### Loop over years in LPJ-GUESS output to create separate netCDF outputs
   for (y in years){
     
+    if (file.exists(file.path(outdir, paste(y,"nc", sep=".")))) {
+      next
+    }
```
Comment (araiho) [inline models/lpjguess/R/model2netcdf.LPJGUESS.R]: Do you need an else statement there? Maybe print no file or something so it shows up in the logfile.txt? Not a big deal though.

**Comment by araiho** [comment]
Looks good to me! It'll be really cool to have LPJ GUESS in pecan.

File: models/lpjguess/R/model2netcdf.LPJGUESS.R (hunk 2 above)
Comment (istfer) [inline models/lpjguess/R/model2netcdf.LPJGUESS.R]: actually this is a subset of LPJ-GUESS outputs, I only went with the ones I was sure that they are an exact match with MsTMIP variables. Also these are only the monthly ones, there are annual outputs too. I was planning to generalize everything in the end, would that be OK?

File: models/lpjguess/NAMESPACE (hunk 1 above)
Comment (mdietze) [inline models/lpjguess/NAMESPACE]: @araiho if you don't export a function it's not available to users when you load the package, but rather can only be used internally by other functions in the package

**Pull request merged**